modules/appeals_api/app/services/appeals_api/pdf_construction/supplemental_claim/v2/form_data.rb

```diff
@@ -102,51 +88,29 @@ def veteran_ssn_last_four
             veteran.ssn[5..8]
           end
 
-          def preferred_phone
+          def signing_appellant_phone
```

Comment:
I typed out a whole rant about keeping the method names more closely in line with the visual PDF field names and I'm glad I double checked myself because `preferred_phone` came from the NOD telephone field while the SC field doesn't mention "preferred" at all 🤯 

While `signing_appellant_phone` (as a name) doesn't map super closely with the pdf field, I do like that it makes it obvious that its value could change depending on who is filling it out. Mostly keeping this comment here because I put so much effort into my unnecessary rant 😉

Reply:
Yeah. We use preferred_phone in other appeals because of the form structure and language on the form, as you mention. I know our forms share so much in common, but I think that's what causes the confusion at times. I would like to see us be really purposeful about our naming and structure based on each forms purpose/data.